### ISSUE TITLE
Unable to run grammar tests

### ISSUE THREAD

**Issue by sigpoggy** [kareman/FootlessParser#2]
I'm running xcode 7 beta 6 - swift2.0 branch
Using the ios scheme it says can't use that device
Using the mac scheme it says build failed
Using the grammar scheme I get 45 errors

**Comment by kareman** [comment]
@sigpoggy Okay that is strange. I just downloaded the project from this GitHub repository and all 3 schemes built and all unit tests pass on my Mac. Could you post the error messages from the Mac and grammar schemes here?

BTW I just pushed a fix for one build error in the grammar scheme. But that presumably leaves you with 44.

**Comment by sigpoggy** [comment]
Just downloaded the latests version and tried again. When I switched to the grammar scheme, this time I got 21 errors. As I got ready to send them to you I noticed they were all undefined symbols in the CSV,swift file. Building and running a grammar test cleared the errors. It must just be xcode was not indexing completely with the change of schemes.
The previous download does have a persistent error, but everything seems to be good now.
Thanks for sharing your work on this project.
